src/table/src/SearchTableHeaderCell.js

```diff
@@ -21,7 +21,7 @@ const invisibleInputClass = css({
   }
 }).toString()
 
-const SearchTableHeaderCell = memo(props => {
+const SearchTableHeaderCell = memo(forwardRef((props, forwardedRef) => {
```

Comment:
Tiniest of non-blocking nits, I think a lot of the codebase just passes in `ref` as this argument and then we can just do `ref={ref}` later on. But obviously non-blocking by any means